db/migrate/20230503114529_add_metadata_to_questionnaires.rb

```diff
@@ -0,0 +1,5 @@
+class AddMetadataToQuestionnaires < ActiveRecord::Migration[6.1]
+  def change
+    add_column :questionnaires, :metadata, :string
```

Comment:
In Sandbox & QA where we're experimenting with questionnaires of different countries, it becomes a pain to keep track of which questionnaire is which by having to go through entire layout.
I have added a metadata column solely for devloper convenience when juggling between different forms of same type.

Now, I can simply run `Questionnaire.select(:id, :is_active, :dsl_version, :questionnaire_type, :metadata)` and set them to active/inactive.

Reply:
We could consider sending the name to the client also, don't need to remove it. The client can choose to use/not use it.

Reply:
Changed `metadata` to `description`. Refraining from modifying API contract, because that seems unnecessary atm. Client does have an ID for referencing a questionnaire for debugging purposes.